magicblock-account-cloner/src/account_cloner.rs

```diff
@@ -71,6 +71,8 @@ pub enum AccountClonerUnclonableReason {
     DoesNotHaveDelegatedEscrowAccount,
     DoesNotAllowEscrowedPda,
     DoesNotAllowFeepayerWithEscrowedPda,
+    /// The account does not exist on-chain (RPC returned empty/system default)
```

Comment:
Did you verify that the RPC returns this? I thought it returns some user error if the account is not found or `None` [see here](https://docs.rs/solana-client/latest/solana_client/rpc_client/struct.RpcClient.html#method.get_account_with_config).